### ISSUE TITLE
[High]: Chrome+Jaws : Screen reader: Role are not defined for controls.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#8406]
## Action Performed:
1. Using Chrome+JAWS, open URL: staging.new.expensify.com
2. Navigate to Settings > Create new Workspace > Search for the created workspace > Details.
3. Navigate to "back (<)", "Members", "Settings", "Invite", etc. controls using the screen reader.
4. Observe the screen reader announcement.

## Expected Result:
When a role is defined appropriately for the interactive element, the screen reader user will interact with the element effectively.

## Actual Result:
Role is not defined for the "back (<)", "Members", "Settings", "Invite", etc. controls. This made it difficult for screen reader users to access the functionality associated with it. Additionally, the label is not defined for the mentioned interactive element in the source code of the page.

**other occurrences:**
>Same issue repro on PR #8065, #8103, #8476, #8409, #9118, #9170, #8878, #9202
>Same issue repro on PR #8760, #8873- Roles are not defined for send and next controls.

## Workaround:
Yes

## Area issue was found in:
Details

## Failed WCAG checkpoints
4.1.2, 1.1.1

## User impact:
Screen reader users will not be made aware that these elements are buttons.

## Suggested resolution:
Ensure that all the page functionality is available for all users irrespective of the device being used.
Apply the following changes:
- Specify role="button" for the 'div' element containing the "back (<)", "Members", "Settings", "Invite", etc. controls.
- Provide aria-label value such as "back" for the 'div' element containing the "back (<)" control.

## Platform:
- Web
- Desktop App
- Mobile Web

**Version Number:** 1.1.49-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Reference link:** https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
**Issue reported by:** Sumit

![#7957_Role and label not defined for back control](https://user-images.githubusercontent.com/98817464/161117999-a854537a-76b2-4340-8d4d-d0f83552e12f.png)
